### ISSUE TITLE
webcord: 3.8.6 -> 3.9.3

### ISSUE THREAD

**Issue by huantianad** [fufexan/webcord-flake#10]
Request for a version bump.

**Comment by fufexan** [comment]
The electron version is new enough in Nixpkgs and dream2nix for 3.9.2. However when trying to build it, some package lookups fail. Feel free to try building it yourself if you think you can fix it.

**Comment by archer-65** [comment]
> The electron version is new enough in Nixpkgs and dream2nix for 3.9.2. However when trying to build it, some package lookups fail. Feel free to try building it yourself if you think you can fix it.

There has been a change in dream2nix.

```nix
  dream = genSystems (system: dreamlib.${system}.makeOutputs {source = webcord;});
```
Now is 
```nix
  dream = genSystems (system: dreamlib.${system}.dream2nix-interface.makeOutputs {source = webcord;});
```

However, the build fails with:
```bash
error: builder for '/nix/store/18b4vj7ylma08kwgmxafy0sf7c0kqqnf-webcord-3.9.2.drv' failed with exit code 1;
       last 10 log lines:
       > > webcord@3.9.2 build
       > > tsc
       >
       > error TS2688: Cannot find type definition file for 'keyv'.
       >   The file is in the program because:
       >     Entry point for implicit type library 'keyv'
       >
       >
       > Found 1 error.
       >
       For full logs, run 'nix log /nix/store/18b4vj7ylma08kwgmxafy0sf7c0kqqnf-webcord-3.9.2.drv'.
error: 1 dependencies of derivation '/nix/store/69iqjrrjmkvrv5mmnxm21mlmbl0f2d2h-webcord-3.9.2-wrapped.drv' failed to build
```

Full logs here: https://pastebin.com/nH9dK2Yy

**Comment by fufexan** [comment]
Yes, that was the error I've got before as well. I am no Node expert, so I don't know how to fix it.

**Comment by archer-65** [comment]
> Yes, that was the error I've got before as well. I am no Node expert, so I don't know how to fix it.

Me neither :(

**Comment by fufexan** [comment]
Updated now, seems like the issue was resolved in 3.10.1.
